### ISSUE TITLE
how to finetune Bart to summarization task of my own?

### ISSUE THREAD

**Issue by RiverTre** [ohmeow/blurr#35]
Hi Blurr, Thank you for your work first.

My question is about [blurr/nbs/02zc_modeling-seq2seq-summarization.ipynb]

Question one:
I have read through this notebook and  could I use the same codes to finetune Bart on my own academic publication dataset?
I mean, after converting my data into the format of cnndm_sample.csv,  Is it right that I only need to change {the csv file location} and {params of the function learn.fit_one_cycle()} ?  

Question two:
If I only need to change the params of learn.fit_one_cycle(), the first param is epoch, and I could change epoch to finetune different models and compare their performance. How about the lr_max param and could I use 4e-5 also? 
About the epoch, do you have suggestions?(I have about 1,000,000 articles of full texts and their abstraction)

**Comment by ohmeow** [comment]
See my article at [here](https://ohmeow.com/posts/2020/05/23/text-generation-with-blurr.html) for a walk-thru of summarization with blurr.  To answer your questions:

1. Yes, you can use all the code as it with a properly formatted dataset.

2. You'll want to use `lr_find` both before and after freezing/unfreezing your model to get an idea of how to set the learning rate.  The best resource here for you is the fastai book and this [lecture 6](https://course.fast.ai/videos/?lesson=6) from the latest fastai course.  I don't have any particular recommendations as of yet, but if you watch the lecture you'll get a good feel for how I'd determine what LRs to use.

**Comment by RiverTre** [comment]
Thank u so much for your suggestions ！
